nemo_megatron_launcher/launch_scripts/nemo_megatron/core/logger.py

```diff
@@ -0,0 +1,50 @@
+import logging
```

Comment:
I tend to keep this. Nemo megatron launcher is supposed to work with virtual env on login node. The launcher itself doesn't have dependency on NeMo / Apex etc. because it only serves as an entry point to create the sbatch script and yaml files.